### DISCUSSION TITLE
Trouble in sign out from keycloak deployed to kubernetes pod

### DISCUSSION THREAD

**Discussion by rssaini01** [keycloak/keycloak#21516] [Archive]
After upgrading from Keycloak 15 to 20, I encountered an error related to the sign-out process. So, I forcefully performed a sign-out for a user from the Sessions page in the Admin Console. After the user refreshed the page, they encountered the following error.

[![enter image description here][1]][1]

Despite the expected behavior of redirecting the user to the login page, this action is not occurring. Users are required to manually remove the cookies in order to access the login page.

The Keycloak server is deployed on a Kubernetes cluster using the [Keycloak Helm chart][2].

  [1]: https://i.stack.imgur.com/BmzXo.png
  [2]: https://github.com/codecentric/helm-charts/tree/master/charts/keycloak

**Comment by vmuzikar** [comment]
@rssaini01 If you are able to provide a reproducer, feel free to open a bug report. Thanks.
